.docker/Dockerfile-py3.ckan

```diff
@@ -11,7 +12,7 @@ ENV CKAN_INI=/app/ckan/default/production.ini
 WORKDIR "${APP_DIR}"
 
 ENV DOCKERIZE_VERSION v0.6.1
-RUN apk add --no-cache curl build-base git libxml2-dev libxslt-dev postgresql postgresql-client postgresql-dev \
+RUN apk add --no-cache curl build-base git libffi-dev libxml2-dev libxslt-dev postgresql postgresql-client postgresql-dev \
```

Comment:
interesting https://sourceware.org/libffi/

Reply:
It's not really needed for this particular extension, it's more for ckanext-harvest, but we may as well make our containers as consistent as possible.